wrapanapi/virtualcenter.py

```diff
@@ -442,30 +449,39 @@ def start_vm(self, vm_name):
     def stop_vm(self, vm_name):
         self.wait_vm_steady(vm_name)
         if self.is_vm_stopped(vm_name):
-            self.logger.info(" vSphere VM %s is already stopped" % vm_name)
+            self.logger.info("vSphere VM %s is already stopped" % vm_name)
             return True
         else:
-            self.logger.info(" Stopping vSphere VM %s" % vm_name)
+            self.logger.info("Stopping vSphere VM %s" % vm_name)
             vm = self._get_vm(vm_name)
             if self.is_vm_suspended(vm_name):
-                self.logger.info(
-                    " Resuming suspended VM %s before stopping." % vm_name
-                )
+                self.logger.info("Resuming suspended VM %s before stopping." % vm_name)
```

Comment:
thought so, didn't want to mess with it too much and the rest of the statements in this file are this format. 

I'll update them all.